### ISSUE TITLE
Question about this project.

### ISSUE THREAD

**Issue by kwontae** [northwesternmutual/grammes#12]
This may be a silly question, but we can use our cosmos DB with gremlin API instead of janus that you guys use in the example, or can we not?

**Comment by damienfamed75** [comment]
@kwontae sorry for late response. 
So... I initially was going to have those a part of this project, but since we don't use Grammes anymore within the company, that functionality was left out.

Theoretically, you should be able to get responses from any Gremlin server using the `ExecuteQuery` function in the Client, but you cannot parse them into the `Vertex` structure for example.

If you wish you help develop that functionality though it'd be more than appreciated to help out Grammes.

**Comment by kwontae** [comment]
@damienfamed75  I would love to learn more about it! I think this is a pretty cool project. If you have time, I would love to connect via slack or something and get a better understanding of what I can do!

**Comment by damienfamed75** [comment]
@kwontae Yeah I'm in the official Golang bridge Slack as `@Damien Stamates`. You're more than welcome to reach out to me.
